src/con_utils.ts

```diff
@@ -304,6 +338,20 @@ function parseConnectDsnAndArgs({
         delete parsedQ.password;
       }
 
+      if ("tls_cert_file" in parsedQ) {
```

Comment:
Hm, using `in` is an antipattern in JS. @fmoor while here, can you add a separate commit making `parsedQ` a `Map` and using `.has()` instead of `in`?

Reply:
PR https://github.com/edgedb/edgedb-js/pull/119

Reply:
changed